nekoyume/Assets/_Scripts/Blockchain/ActionHandler.cs

```diff
@@ -30,61 +26,17 @@ public abstract class ActionHandler
 
         public abstract void Start(ActionRenderer renderer);
 
-        protected static bool HasUpdatedAssetsForCurrentAgent<T>(ActionEvaluation<T> evaluation)
-            where T : ActionBase
-        {
-            if (States.Instance.AgentState is null)
-            {
-                return false;
-            }
-
-            return evaluation.OutputState.Delta.UpdatedFungibleAssets.Any(tuple =>
-                tuple.Item1.Equals(States.Instance.AgentState.address));
-        }
-
-        protected static bool HasUpdatedAssetsForCurrentAvatar<T>(ActionEvaluation<T> evaluation)
-            where T : ActionBase
-        {
-            return States.Instance.CurrentAvatarState is not null &&
-                   evaluation.OutputState.Delta.UpdatedFungibleAssets.Any(tuple =>
-                       tuple.Item1.Equals(States.Instance.CurrentAvatarState.address));
-        }
-
-        protected static bool ValidateEvaluationForCurrentAvatarState<T>(ActionEvaluation<T> evaluation)
-            where T : ActionBase
-        {
-            if (!(States.Instance.CurrentAvatarState is null))
-            {
-                var avatarAddress = States.Instance.CurrentAvatarState.address;
-                var addresses = new List<Address>
-                {
-                    avatarAddress,
-                };
-                string[] keys =
-                {
-                    LegacyInventoryKey,
-                    LegacyWorldInformationKey,
-                    LegacyQuestListKey,
-                };
-                addresses.AddRange(keys.Select(key => avatarAddress.Derive(key)));
-                return addresses.Any(a =>
-                    evaluation.OutputState.Delta.UpdatedAddresses.Contains(a));
-            }
-
-            return false;
-        }
-
```

Comment:
Is it correct changes?

Reply:
Yes. Previously, Headless had a bug with filtering by subscribed addresses when publishing render data. It causes sending all rendering events to all clients.

In this [PR](https://github.com/planetarium/NineChronicles.Headless/pull/2258/files#diff-85517736648a9d4f67edbc271b7886433340a2dce678083bf547014fa132d535), we fix that bug and no more filtering needed at the Unity.